packages/react/src/components/Authenticator/SignUp/SignUp.tsx

```diff
@@ -18,8 +18,6 @@ export function SignUp() {
     },
   } = components;
 
-  console.log({ components });
```

Comment:
How could you 😈

Reply:
Maybe a lint rule to prevent this in prod build?

Reply:
Totally agree – https://github.com/aws-amplify/amplify-ui/issues/769